src/common/utils.js

```diff
@@ -293,11 +293,21 @@ const noop = () => {};
 const logger =
   process.env.NODE_ENV !== "test" ? console : { log: noop, error: noop };
 
+// Cache settings.
+const CARD_CACHE_SECONDS = 14400;
+const ERROR_CACHE_SECONDS = 600;
```

Comment:
Let's update this to 30min? Currently it's 10. And if PATs are exhausted that means Vercel will serve fresh responses every 10 minutes, but PATs will only restore after 30min or 1hour.

Changing it to 30min will reduce the load on vercel.

Reply:
@anuraghazra GitHub does restore the individual PATs after 1 hour. However, the frequency at which PATs are refreshed depends on when the PAT was depleted.

![image](https://github.com/anuraghazra/github-readme-stats/assets/17570430/503af8ba-4c75-42a0-b1b8-0a39169d97a2)

As a result, the 10 minutes could catch a refreshed token. However, if you think the load of Vercel should be decreased, I am happy to increase it to `1800`.

Reply:
Okay i think we can keep it as is, load on Vercel is not a huge deal right now.